test/orbit/import/rfid_test.exs

```diff
@@ -0,0 +1,90 @@
+defmodule Orbit.Import.RfidTest do
+  alias Orbit.BadgeSerial
+  alias Orbit.Employee
+  use Orbit.DataCase
+  alias Orbit.Import.Rfid
+  alias Orbit.Import.RfidWorker
+  import Orbit.Factory
+
+  describe "worker" do
+    test "downloads and parses badge serials" do
+      insert(:employee, %{badge_number: "1234", badge_serials: []})
```

Comment:
Could "1234" be a module constant?

Reply:
It maybe could, but not all of the tests are quite the same in terms of what IDs they use (note that there's one test that has two IDs).